src/App.scss

```diff
@@ -177,6 +181,10 @@ table tr:nth-child(even){
   margin-left: 17px;
 }
 
+.margin-left-medium {
+  margin-left: 10px;
+}
```

Comment:
should we use styled components for the elements using these?

Reply:
We totally should, have done so.